prisma/schema.prisma

```diff
@@ -1,12 +1,205 @@
-// This is your Prisma schema file,
-// learn more about it in the docs: https://pris.ly/d/prisma-schema
+generator client {
+  provider = "prisma-client-js"
+}
 
 datasource db {
-  provider          = "postgresql"
-  url               = env("DATABASE_URL")
-  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
 }
 
-generator client {
-  provider = "prisma-client-js"
+model admin {
```

Comment:
Please change model names to follow PascalCase, singular. This can be done by changing `admin` to `coupon_users` to `CouponUser`. Then we add `@@mapping(name: "admin")` and `@@mapping(name: "coupon_users"` at the end of each respective model.

Please see https://www.prisma.io/docs/concepts/components/prisma-schema/data-model for reference and also the `schema.prisma` file in dancefest-web repository